lms/djangoapps/discussion_api/api.py

```diff
@@ -634,6 +647,7 @@ def update_thread(request, thread_id, update_data):
     # Only save thread object if some of the edited fields are in the thread data, not extra actions
     if set(update_data) - set(actions_form.fields):
         serializer.save()
+        thread_edited.send(sender=None, user=request.user)
```

Comment:
Maybe a simpler thing would be to only count modifications/signals sent by members of the team? That kind of filtering could happen on the model, and we wouldn't have to change the format of the signal as much.